controllers/storagecluster/storageconsumer.go

```diff
@@ -109,6 +111,16 @@ func (s *storageConsumer) ensureDeleted(_ *StorageClusterReconciler, _ *ocsv1.St
 	return ctrl.Result{}, nil
 }
 
+func getExternalStorageClassBlaclistSelector() labels.Selector {
```

Comment:
This is used for snapshotClasses and groupSnapshotClasses as well. Can we use a better name for this?

Reply:
getExternalStorageClassesBlaclistSelector? or getExternalClassesBlaclistSelector?